smurfs/src/components/PostSmurf.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect }from 'react'
+import axios from 'axios'
+import { connect } from 'react-redux'
+
+import { postSmurf} from '../actions'
+
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import { Container } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+    container: {
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap'
+      },
+      formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120
+      },
+      selectEmpty: {
+        marginTop: theme.spacing(2)
+      },
+    
+      paper: {
+        marginTop: theme.spacing(8),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center'
+      },
+      form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(3)
+      },
+      submit: {
+        margin: theme.spacing(3, 0, 2),
+        backgroundColor: '#017dc7',
+        '&:hover': {
+          backgroundColor: '#de5916'
+        }
+      }
+  }));
+
+const PostSmurf = (props) => {
+      const classes = useStyles();
+      const [newSmurf, setNewSmurf] = useState({
+        name: '',
+        age: '',
+        height: ''
+      });
+      const [smurf, setSmurf] = useState({});
+
+      useEffect(() => {
+        axios
+            .get(`http://localhost:3333/smurfs`)
+            .then(res => {
+                console.log(res);
+                setSmurf(res.data);
+            })
+            .catch(err => {
+                console.log('ERROR in GET post api', err.response);
+            })
+        }, [])
```

Comment:
Because `<SmurfCard/>` calls the action that fetches our smurfs, we don't need this `useEffect()` in our form '' '